src/PoissonSolvers/FFTOpenPoissonSolver.hpp

```diff
@@ -107,16 +107,18 @@ void unpack(const ippl::NDIndex<3> intersect,
 }
 
 template <typename Tb, typename Tf, unsigned Dim>
-void solver_send(const ippl::NDIndex<Dim> intersection, const ippl::NDIndex<Dim> ldom, int nghost, int id, int i, Kokkos::View<Tf***>& view,
-            ippl::detail::FieldBufferData<Tb>& fd, std::vector<MPI_Request>& requests){
+void solver_send(const ippl::NDIndex<Dim> intersection, const ippl::NDIndex<Dim> ldom, int nghost,
+                 int id, int i, Kokkos::View<Tf***>& view, ippl::detail::FieldBufferData<Tb>& fd,
+                 std::vector<MPI_Request>& requests) {
     using memory_space = Kokkos::View<Tf***>::memory_space;
 
     requests.resize(requests.size() + 1);
 
     ippl::Communicate::size_type nsends;
     pack(intersection, view, fd, nghost, ldom, nsends);
 
-    ippl::Communicate::buffer_type<memory_space> buf = ippl::Comm->getBuffer<memory_space, Tf>(IPPL_VICO_SEND + id * 8 + i, nsends);
+    ippl::Communicate::buffer_type<memory_space> buf =
+        ippl::Comm->getBuffer<memory_space, Tf>(IPPL_VICO_SEND + id * 8 + i, nsends);
```

Comment:
please explain the 8 in general all such constants should ALWAYS be motivated in a comment

Reply:
I actually had this question as well. Why do we need an 8 if the buffer tag will be unique anyway with the combination of `id` (corresponding to the spatial dimension in the vico sends) and `i` (corresponding to the rank)?